README.md

```diff
@@ -92,7 +92,7 @@ sudo service apache2 reload
     
 6. Go to http://localhost/installdb.php and follow the instructions to finalize LORIS installation, then restart apache.
 
-If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli.requirements.php) and also [this](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password-compatible-connectors).
+If you use MySQL 8 and PHP 7.3 (instead of PHP 7.4), please read [this link](https://www.php.net/manual/en/mysqli.requirements.php) and also [this](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password-compatible-connectors).
```

Comment:
@AlexandraLivadas I think this note is worth adding to #6511. If we do so, we can probably remove this line from the main README.

Reply:
Yes, this line looks strange in the README.md, the note should have a better place, and the problem of MySQL 8.0 is much less a problem now as before vs PHP7.3. @johnsaigle

Reply:
Should I add this to #6510, which is the Ubuntu install markdown? Or do you think it should also be added to the troubleshooting guide (#6511) @johnsaigle ?

Reply:
Yeah I think the troubleshooting guide is better. Since this isn't specific to Ubuntu as far as I can tell, it should be in the cross-platform document.